controllers/roadmap-site/challengeController.js

```diff
@@ -0,0 +1,65 @@
+const logger = require("../../utils/logger");
+const challengeQuery = require("../../models/roadmap-site/challenges");
+const subscribeToChallengeQuery = require("../../models/roadmap-site/subscribe-challenge");
+
+const errorAdminString = "Something went wrong, please try again/contact admin";
+
+const sendChallengeResponse = async (req, res) => {
```

Comment:
Add JSDOC and annotations for documentation above every controller function.

Reply:
Done